### DISCUSSION TITLE
Is it intentional that the tooltip does not close automatically in touch mode?

### DISCUSSION THREAD

**Discussion by mrclrchtr** [mantinedev/mantine#6846] [Q&A]
As already asked in the headline: Is it intentional that the tooltip does not close automatically in touch mode?

I am trying to achieve that the “copied” tooltip only appears briefly on the copy button.

**Comment by Kenzo-Wada** [comment]
I'm not sure I understand correctly what you mean by “touch mode”, but you can control the time it takes to close with a prop called closeDelay, how about that?
Or there is also a hack to set the “opened” props to true for a few seconds with useState and setTimeout!

**Reply by mrclrchtr** [reply]
I mean the mobile or smartphone mode. In other words, in the mode in which control is done by hand/touch.

And then set touch to true: https://mantine.dev/core/tooltip/#change-events
You can test the behavior in mobile mode on the mantine docs page.

Yes, that's exactly how I would do the workaround, but I'm just wondering whether it's intended that way.
